src/attribute-editor/__tests__/attribute-editor.test.tsx

```diff
@@ -79,6 +79,26 @@ const expectRowErrorTextContent = (wrapper: AttributeEditorWrapper, rowIndex: nu
   });
 };
 
+function TestComponent(propOverrides: Partial<AttributeEditorProps<Item>> = {}) {
```

Comment:
This setup is needed to test the new prop since it relies on re-render on state-change.